store/proofs/create.go

```diff
@@ -0,0 +1,121 @@
+package proofs
+
+import (
+	"errors"
+	"fmt"
+	"sort"
+
+	"github.com/tendermint/tendermint/crypto/merkle"
```

Comment:
create.go seems not a good name.

Reply:
I just copied the files from tendermint, so I think we can just keep these files for audit or further changes.